docs/post_installation/managing.md

```diff
@@ -129,30 +133,30 @@ Your node's ledger accumulates many messages, which uses a significant disk capa
   },
 ```
 
-### Snapshot pruning
-During a snapshot, messages may be deleted from the ledger if they were confirmed by an old milestone. In other words, the term `pruning` means the deletion of the old history from the node database:
-* To enable pruning, set the `pruning.milestones.enabled` or `pruning.size.enabled` key to enabled
-* The `pruning.milestones.maxMilestonesToKeep` defines how far back from the current confirmed milestone should be pruned
-* The `pruning.size.targetSize` defines the maximum database size, old data will be pruned.
+### Snapshot Pruning
+During a snapshot, Hornet may delete messages from the ledger if they were confirmed by an old milestone. In other words, the term _pruning_ means the deletion of the old history from the node database.  
+
+* If you want to enable pruning, you should set the `pruning.milestones.enabled` or `pruning.size.enabled` keys to _enabled_.
+* The `pruning.milestones.maxMilestonesToKeep` defines how far back from the current confirmed milestone should be pruned.
+* The `pruning.size.targetSize` defines the maximum database size.  Old data will be pruned.
 
 There are two types of snapshots:
 
-**Delta snapshot**
-A delta snapshot points to a specific full snapshot, ie. delta snapshot consists of the changes since the last full snapshot.
+#### Delta snapshots
+A delta snapshot points to a specific full snapshot, this means a delta snapshot consists of the changes since the last full snapshot.
```

Comment:
```suggestion
A delta snapshot points to a specific full snapshot, meaning a delta snapshot consists of the changes since the last full snapshot.
```